hw/ip/hmac/data/hmac.hjson

```diff
@@ -3,9 +3,10 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name:               "hmac",
+  human_name:         "HMAC Accelerator",
   one_line_desc:      "Accelerator for SHA-256-based keyed hash message authentication code and the SHA-256 hash function",
   one_paragraph_desc: '''
-  The OpenTitan HMAC hardware block is a keyed hash based message authentication code generator using [SHA-256][nist-fips-180-4] to check the integrity of an incoming message and optionally a signature signed with the same secret key.
+  The OpenTitan HMAC accelerator is a keyed hash based message authentication code generator using [SHA-256][nist-fips-180-4] to check the integrity of an incoming message and optionally a signature signed with the same secret key.
```

Comment:
General question: should we also align the capitalization? Here the human name is `HMAC Accelerator` but in the text we write `HMAC accelerator` (some for other accelerators and I2C interface, key manager etc.) whereas for `SPI Device` and `SPI Host` we also write `SPI Device` and `SPI Host` in the text (similarly USB Device and System Reset Controller)

Reply:
Thanks for this question, @vogelpi. My gut feeling is that capitalization of the block name in sentences is not worth standardizing and enforcing. It's easy to capitalize them consistently in the `human_name` field, so names are formatted uniformly whenever content is generated based on that field (titles, list entries, ...). Doing that for all text is a lot of manual effort, though, for which I see little benefit. I think it's much more important that we use those names consistently than that we capitalize them consistently.

Though we could do a one-time effort now for the one-line and one-paragraph descriptions, but I would stop after that.  WDYT?

Reply:
Aligning the one-line and one-paragraph summaries sounds great to me. I agree that doing that for all text is out of scope.

Reply:
Goes along with what was done for Entropy Source. Ultimately tooling can help with getting this consistent everywhere but in this effort just manually capitalising makes sense to me.
`HMAC Accelerator`